### ISSUE TITLE
The AP is  so low and increase slowly in the early process on COCOtrainval35k.

### ISSUE THREAD

**Issue by kongqiankun** [fizyr/keras-retinanet#642]
This is my training process on COCOtrainval35k with using Resnet101. I dom't know why the ap is so low and increase slowly in the early epoch. The training batchsize is 1,steps is 10000 every epoch, and the validation is 5k.
![image](https://user-images.githubusercontent.com/24432040/44440874-b1e95280-a5fc-11e8-8777-380c76a1cdf0.png)

![image](https://user-images.githubusercontent.com/24432040/44440882-b9a8f700-a5fc-11e8-9704-3bbfdb6e3097.png)

![image](https://user-images.githubusercontent.com/24432040/44440891-be6dab00-a5fc-11e8-9184-477cc66df6c3.png)

**Comment by yhenon** [comment]
You're going to need to train a lot more. You've done 100k iterations at batch size 1, which is not even 1 pass over all the data. I'd recommend training about 10x more.

**Comment by kongqiankun** [comment]
now , I have trained 26 epochs, and the AP is 25.5. Thanks, I will do it more epochs.  Besides, I think I should set the epochs in 100 at least according to  your suggestion, I am looking forward to the next result.

**Comment by kongqiankun** [comment]
now, I have done 76 epochs, but the Ap is 26.8.  And now the loss is very low(0.8). So I think the Ap won't reach the 37.8 showed by the originanl paper's result. What should I do now? I am blank. and I don't know the reason. Please.
